EasyPost/Services/BatchService.cs

```diff
@@ -103,13 +106,163 @@ public async Task<Batch> CreateAndBuy(BetaFeatures.Parameters.Batches.Create par
 
         // TODO: Add GetNextPage function when Batches are sorted newest to oldest.
 
+        /// <summary>
+        ///     Add shipments to this batch.
+        /// </summary>
+        /// <param name="parameters">Update shipment parameters.</param>
+        /// <returns>The updated Batch.</returns>
+        [CrudOperations.Update]
+        public async Task<Batch> AddShipments(string id, Dictionary<string, object> parameters)
+        {
+            // parameters = parameters.Wrap("batch");  // TODO: Update docs to remove wrapped "batch" key
```

Comment:
question: if we have a TODO about a project that isn't directly in this code, we should be creating tasks instead of leaving TODOs in a place they'll be forgotten about because it's not related.

Why do we need to remove the wrapped batch key from the docs?

Let's remove this TODO and instead create a more actionable follow up (if necessary)

Reply:
Resolved